extensions/amazon-lambda-rest/runtime/src/main/java/io/quarkus/amazon/lambda/http/LambdaSecurityIdentity.java

```diff
@@ -0,0 +1,64 @@
+package io.quarkus.amazon.lambda.http;
+
+import java.security.Permission;
+import java.security.Principal;
+import java.util.Collections;
+import java.util.Map;
+import java.util.Set;
+
+import io.quarkus.security.credential.Credential;
+import io.quarkus.security.identity.SecurityIdentity;
+import io.smallrye.mutiny.Uni;
+
+public class LambdaSecurityIdentity implements SecurityIdentity {
```

Comment:
You should be able to just use QuarkusSecurityIdentity here, no need for a new class.